src/main/java/catan/controllers/GameController.java

```diff
@@ -61,7 +68,7 @@ public GameIdDetails createNewGame(@RequestParam(value = "token", required = fal
 
         List<GameDetails> gamesToReturn = new ArrayList<GameDetails>();
         for (GameBean game : games) {
-            gamesToReturn.add(new GameDetails(game));
+            gamesToReturn.add(new GameDetails(game, userId));
```

Comment:
btw, do we really need list of resources when retrieving list of games? I think we just need it when player tries to get details of some particular game